benchto-driver/src/main/java/io/trino/benchto/driver/graphite/GraphiteMetricsLoader.java

```diff
@@ -150,8 +153,8 @@ private void addMeanMaxMeasurements(Map<String, double[]> loadedMetrics, List<Me
     {
         Optional<StatisticalSummary> statistics = getStats(loadedMetrics, metricName);
         if (statistics.isPresent()) {
-            measurements.add(Measurement.measurement("cluster-" + metricName + "_max", unit, statistics.get().getMax()));
```

Comment:
won't this make scripts harder, e.g: previously I could just select name in SQL query. Now I will have to join on attributes and do subquery. Seems complicated